polkadot/xcm/xcm-executor/src/lib.rs

```diff
@@ -84,6 +84,7 @@ pub struct XcmExecutor<Config: config::Config> {
 	transact_status: MaybeErrorCode,
 	fees_mode: FeesMode,
 	_config: PhantomData<Config>,
+	assetClaimer: Option<Location>,
```

Comment:
We're using CamelCase for struct names, and snake_case for variable/member names.

```suggestion
	asset_claimer: Option<Location>,
```

Please check
https://github.com/paritytech/polkadot-sdk/blob/master/docs/contributor/CONTRIBUTING.md
and
https://github.com/paritytech/polkadot-sdk/blob/master/docs/contributor/STYLE_GUIDE.md
for more contributor info.

tip: `cargo` can do most of the style formatting for you. Just run `cargo fmt -p <crate-name>` or `cargo fmt --all`.
In our case you should use the nightly version for formatting for some advanced linters we use: `cargo +nightly fmt -p <crate-name>`